contracts/token/superfluid/SuperToken.sol

```diff
@@ -117,6 +153,10 @@ contract SuperToken is UUPSProxiable, SuperfluidToken, ISuperToken {
 	function updateCode(address newAddress) external virtual override {
 		if (msg.sender != address(_host)) revert SUPER_TOKEN_ONLY_HOST();
 		UUPSProxiable._updateCodeAddress(newAddress);
+
+		// this allows us to deploy and set the nft proxy contracts for existing
+		// supertokens that are in the wild only if the nft proxy contracts are unset
+		_deployAndSetNFTProxyContractsIfUnset();
 	}
 
```

Comment:
Please add `operationIncreaseAllowance` and `operationDecreaseAllowance`.
These two functions were added in 1.5.2 release, they were implemented to make `increaseAllowance` and `decreaseAllowance` batchable. This means that if end users try to utilize our Stream Scheduler (can set close-ended streams) or Vesting features, it will break without these two functions.

```
function operationIncreaseAllowance(
    address account,
    address spender,
    uint256 addedValue
)
    external override
    onlyHost
{
    _approve(account, spender, _allowances[account][spender] + addedValue);
}
    
function operationDecreaseAllowance(
    address account,
    address spender,
    uint256 subtractedValue
) 
    external override
    onlyHost
{
    _approve(account, spender, _allowances[account][spender].sub(subtractedValue,
        "SuperToken: decreased allowance below zero"));
}
```

Reply:
added